src/idl_gen_cpp.cpp

```diff
@@ -640,7 +634,7 @@ class CppGenerator : public BaseGenerator {
   // Return a C++ type from the table in idl.h
   std::string GenTypeBasic(const Type &type, bool user_facing_type) const {
     // clang-format off
-    static const char *const ctypename[] = {
+    static const char* const ctypename[] = {
```

Comment:
Here `*` also was moved to the left.
There is a problem if code surrounded by `off/on` formatting tags.

Reply:
Thanks, will fix.